pyproject.toml

```
@@ -37,7 +37,7 @@ classifiers = [
   "Topic :: Text Processing :: Markup",
 ]
 dependencies = [
-  "sphinx~=7.3"
+  "sphinx>=7.3"
```

```suggestion
  "sphinx>=8,<9"
```

just support sphinx 8, otherwise the test infrastructure should be updated to test both 7 and 8

---

done